src/components/Cards/NFTCard/NFTCard.module.css

```diff
@@ -41,6 +41,7 @@
 }
 
 .NFT {
+	padding: 8px;
```

Comment:
Won't need this padding with the cropped images -- [relevant Figma frame](https://www.figma.com/file/0zFJaRNODss8lxwRygUh24/UI-Wilder-Market?node-id=6024%3A240680)